components/tooltip/tooltip.js

```diff
@@ -24,6 +24,17 @@ const contentBorderRadius = 6;
 const contentBorderSize = 1;
 const contentHorizontalPadding = 15;
 
+/* once a user shows a tooltip, ignore delay if they hover adjacent target within this timeout */
```

Comment:
This is some usability extra sugar that Jeff wanted added to tooltips.  The basic idea here is - once a tooltip has been displayed for a target, if the user hovers over another target within 1s of showing the first tooltip, then we should ignore the delay and just show the tooltip immediately.